### DISCUSSION TITLE
Curl no longer shipped with Samsung phones?

### DISCUSSION THREAD

**Discussion by jidanni** [curl/curl#12296] [Q&A]
https://daniel.haxx.se/blog/2018/09/17/the-worlds-biggest-curl-installations says
> curl is bundled with the Android version that Samsung, Xiaomi and OPPO ship

Well, OK, I found 
- /system/bin/nc, but alas no
- /system/bin/curl

at least on my Samsung Galaxy A13 5G.

So maybe curl is no longer being shipped on Samsung phones?

Sorry if it is a billion customer loss. Don't jump off any roofs or anything.

**Comment by bagder** [comment]
I'm not aware of Samsung (or any other Android) ever having shipped the curl tool. They have used libcurl.

**Reply by jidanni** [reply]
Hmm, the blog post talks a lot about libcurl. But by the time it gets down to the Samsung part, it has forgotten to say libcurl and just says curl.

So one thinks it is talking about a standalone binary bin/curl.
